include/core/mimedata/mimedata.hpp

```diff
@@ -59,8 +59,8 @@ namespace Toolbox {
         [[nodiscard]] std::optional<std::string> get_text() const;
         void set_text(std::string_view data);
 
-        [[nodiscard]] std::optional<std::string> get_urls() const;
-        void set_urls(std::string_view data);
+        [[nodiscard]] std::optional<std::vector<std::string>> get_urls() const;
+        void set_urls(std::vector<std::string> data);
```

Comment:
Pass by const reference to reduce allocations.

Reply:
Fixed by [this commit](https://github.com/JoshuaMKW/JuniorsToolbox/pull/36/commits/565d229c848d5514ee703fd7ec29eb76885dcaa1).